ethcore/src/service.rs

```diff
@@ -130,6 +131,31 @@ impl ClientService {
 	pub fn add_notify(&self, notify: Arc<ChainNotify>) {
 		self.client.add_notify(notify);
 	}
+
+	#[cfg(feature="stratum")]
+	/// Runs stratum if feature is on
+	pub fn stratum_probably(cfg: &Option<StratumOptions>, miner: &Arc<Miner>, client: &Arc<Client>) -> Option<Box<miner::NotifyWork>> {
```

Comment:
If the configuration is required for startup, shouldn't the check should occur outside this function?